<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindUSB</class>
 <widget class="QWidget" name="FindUSB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выбор SDR-устройства</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #fff;
}
QLabel{
 background-color: #fff;
}

QPushButton {
  background-color: #0d6efd;
  color: #fff;
  border-radius: 8px;
  border: 1px solid #0d6efd;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}
QPushButton:hover,
QPushButton:focus {
  background-color: #0b5ed7;
  border: 3px solid #9ac3fe;
}
QComboBox{
color: white; 
background-color: #0d6efd;
selection-background-color: lightgreen;
    font: normal normal 27px &quot;Arial&quot;;
}
QComboBox QAbstractItemView {
    background-color: #0d6efd;
    selection-background-color: lightgreen;
    font: normal normal 27px &quot;Arial&quot;;
}
QComboBox QAbstractItemView::item {
    height: 100px;  
    font: normal normal 27px &quot;Arial&quot;;
}
QListWidget
{
font: normal normal 27px &quot;Arial&quot;;
border : 2px solid black;
background-color: #0d6efd;
}

 QListView::item:selected
{
border : 2px solid black;
background : green;
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>790</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>790</width>
        <height>70</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Выберите устройство из списка ниже:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_usb">
      <property name="minimumSize">
       <size>
        <width>790</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>790</width>
        <height>200</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>790</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancel_button_usb">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="update_button_usb">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="continue_button_usb">
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выбрать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
